.github/workflows/publish.yaml

```diff
@@ -121,36 +44,25 @@ jobs:
           fi
 
           echo "$TAG"
+          echo "tag=${TAG,,}" >> ${GITHUB_OUTPUT}
 
-          echo "VERSION=${TAG,,}" >> ${GITHUB_ENV}
-          echo "DOCKER_REGISTRY=${{ env.REGISTRY }}" >> ${GITHUB_ENV}
-          # Lowercase the owner for docker image naming conventions
-          echo "DOCKER_ORG=${OWNER,,}" >> ${GITHUB_ENV}
-          java --version
-
-      # 2.5 - Login against the docker registry
-      - name: Login to registry ${{ env.REGISTRY }}
-        uses: docker/login-action@v2
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Show Generated Tag
+        run: echo ${{ steps.generate.outputs.tag }}
 
-      # 2.6 - Setup QEMU for platform emulation
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      # 2.7 - Setup Docker BuildX for multi platform building
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-
-      # 2.8 - Build and push Docker Images
-      - name: Build Images using BuildX Bake
-        uses: docker/bake-action@v2
-        with:
-          files: ./${{ env.BAKE_FILE }}
-          targets: ${{ matrix.target }}
-          push: true
-          set: |
-            *.cache-from=type=gha,scope=build-${{ matrix.target }}
-            *.cache-to=type=gha,scope=build-${{ matrix.target }},mode=max
+  # Build and Publish all targets associated with specified group
+  bake:
+    needs:
+      - lint
+      - lint-client
+      - test-client
+      - tag-generator
+    uses: darpa-askem/.github/.github/workflows/bake-publish.yml@main
```

Comment:
Do we need to save the workflow under `/.github` folder, when they are already in the `.github` repo?

Reply:
Unfortunately yes, we tried without it and it did not work.